desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -71,25 +71,36 @@ def renew_delegation_token(self, user):
 
 class RazClient(object):
 
-  def __init__(self, raz_url, raz_token, username, service='s3', service_name='cm_s3', cluster_name='myCluster'):
+  def __init__(self, raz_url, raz_token, username, service='s3', service_name='cm_s3', cluster_name='myCluster', operation={}):
```

Comment:
We should not need to add `operation` but prepare the correct request_data instead inside check_access